pkgs/top-level/all-packages.nix

```diff
@@ -283,10 +289,10 @@ with pkgs;
       netrcImpureEnvVars = [ "${varBase}USERNAME" "${varBase}PASSWORD" ];
     };
     fetcherArgs = (if fetchSubmodules
-        then { inherit rev fetchSubmodules; url = "${baseUrl}.git"; }
-        else ({ url = "${baseUrl}/archive/${rev}.tar.gz"; } // privateAttrs)
+        then { inherit rev tag fetchSubmodules; url = "${baseUrl}.git"; }
+        else ({ url = "${baseUrl}/archive/${revOrTag}.tar.gz"; } // privateAttrs)
```

Comment:
Did you forget to escape here?

Reply:
GitHub does not seem to need escaping. GitLab does not need it either but fetchzip does, in order to be able to extract the file.

Reply:
Oh nevermind I see the difference. Isn't there still some escaping necessary, just not of the `/`?